Slingshot.Api/Abstract/Repository.cs

```diff
@@ -146,6 +146,11 @@ public virtual Task<bool> HasScmInfo()
             return Task.FromResult(false);
         }
 
+        public virtual Task WritePullRequestComment(string prId, string comment)
+        {
+            return Task.FromResult(new object());
```

Comment:
currently targeting .net 4.5
might be a good idea to upgrade this project to 4.6

i will create a separate work item for it.